### DISCUSSION TITLE
Google、Facebook 也開始進攻海底電纜

### DISCUSSION THREAD

**Discussion by HSU-YU-JEN** [1101-GS4719-C/Python-Community-GS4719-C#246] [科技新知]
原本大多數的電纜都是電信公司們共同擁有，不過現在如 Google、Facebook、微軟、亞馬遜等都是新電纜的主要投資者，這些矽谷企業也正在積極「卡位」中。
例如，Google 就霸佔了大西洋，計劃於美國、英國和西班牙間鋪設總長超過 1.2 萬公里的海底電纜，預計 2022 年完工；Facebook 則先搶進非洲，長度可以繞地球一周，達到 3.7 萬公里。

參考網址
https://buzzorange.com/techorange/2021/01/28/submarine-cable/

**Comment by HSU-YU-JEN** [comment]
這個是台灣的海底電纜圖
從台灣連接到中國福州的電纜 Taiwan Strait Express-1（TSE-1）長 260 公里，由中國聯通、遠傳電信、台灣大哥大、中華電信、中國移動、台灣國際纜網通信共同擁有。
![image](https://user-images.githubusercontent.com/90545550/147751527-cd7d6a06-6c30-4e20-bf17-7d3878857ed8.png)

**Reply by HSU-YU-JEN** [reply]
而且在今年8月底左右，因為中美關係緊繃，許多矽谷企業如Google、Facebook 都放棄了一項連結美國與香港的跨太平洋海底電纜建設計畫，選擇了台灣，這其實也讓台灣在亞洲網路的重要性因此提升，形成另類的護國戰略。

**Comment by 107602505** [comment]
看完發現自己對海底電纜並沒有太多的了解，一直以為通網路訊是透過衛星傳遞，沒想到原來海底電纜才是承擔整個全球網路的通信，後來再去查一些相關報導，覺得這篇將海底電纜介紹的蠻仔細的~
https://www.techbang.com/posts/80762-the-story-of-the-submarine-cable

**Reply by 109802503gloria** [reply]
這篇文章真的超詳細，又長見識了

**Reply by Jie2222** [reply]
感謝同學分享，以前真的不知道海底電纜是網路通信最優的選擇，目前仍無法為衛星所取代

**Comment by zixunn** [comment]
看了這篇這麼詳細的文，我才知道原來衛星佔據的國際網路傳輸量比電纜少那麼多，也感謝另一篇補充電纜知識的文章分享，第一次知道原來海底電纜之多，其總長可以繞地球30圈。

**Comment by MAKERRRofKEVIN** [comment]
感謝同學分享，讓我一直一來的對於國際通訊橋樑的好奇有了一些解惑，過去以為都是透過衛星或無線基地台傳訊的，沒想到海底電纜的重要性與工程有這麼大

**Comment by syt-110** [comment]
之前系上老師說海底電纜鋪設與地形及海水流速和出海砂石量有關，但沒有細查，感謝同學分享!
